src/test/unit/controller/MultipleRespondentCheckController.test.ts

```diff
@@ -0,0 +1,89 @@
+import MultipleRespondentCheckController from "../../../main/controllers/MultipleRespondentCheckController";
```

Comment:
Could you please tidy up spacing in this file a little bit?

Reply:
refactored and have redone tests as well